docs/source/guide/export.md

```diff
@@ -1,33 +1,103 @@
 ---
-title: Export results
+title: Export results from Label Studio
 type: guide
-order: 105
+order: 402
 ---
 
-Your annotations are stored in [raw completion format](#Completion-format) inside `my_project_name/completions` directory, one file per labeled task named as `task_id.json`.
+## Export data from Label Studio
 
-You can optionally convert and export raw completions to a more common format by doing one of the following:
+Export your completed annotations from Label Studio. Label Studio stores your annotations in a raw JSON format in the SQLite database backend or whichever cloud or database storage you specify as target storage, with one file per labeled task named as `task_id.json`. 
 
-- From [/export](http://localhost:8080/export) page by choosing target format
-- Applying [converter tool](https://github.com/heartexlabs/label-studio-converter) to `my_project_name/completions` directory
-- By using [Export API](#Export-using-API)
+You can convert the raw JSON completed annotations stored by Label Studio into a more common format and export that data in several different ways:
 
-## Basic format
+- Export from the Label Studio UI on the [/export](http://localhost:8080/export) page.
+- Run the relevant [converter tool](https://github.com/heartexlabs/label-studio-converter) on the directory of completed annotations using the command line or Python.
```

Comment:
there is no directory of completed annotations since 1.0.0. (but it's true from older versions). It could be a single exported JSON file though